jujugui/static/gui/src/app/app.js

```diff
@@ -639,6 +639,47 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Sends the discharge token via POST to the storefront. This is used
+      when the GUI is operating in GISF mode, allowing a shared login between
+      the GUI and the storefront.
+
+      @method _sendGISFPostBack
+    */
+    _sendGISFPostBack: function() {
+      const dischargeToken = window.localStorage.getItem('discharge-token');
+      if (!dischargeToken) {
+        console.error('no discharge token in local storage after login');
```

Comment:
I don't think so; with the code moved here this should only happen with a completely failed login, so the user is going to see a failed login and get the login window again anyway.

It's possible this error can never be reached, but if it is we'll want to be able to see this in the console.